creator/data_reviews/models.py

```diff
@@ -0,0 +1,151 @@
+import uuid
+from django.db import models
+from django.contrib.auth import get_user_model
+from django_fsm import FSMField, transition
+
+from creator.fields import kf_id_generator
+from creator.studies.models import Study
+from creator.files.models import File
+
+User = get_user_model()
+
+
+def data_review_id():
+    return kf_id_generator("DR")
+
+
+class State(object):
```

Comment:
Would this also work as an Enum?

Reply:
Yes, but then I have to type out for example, `State.NOT_STARTED.value`, every time I want to access the value of the enumerated item